app/src/main/java/com/kaamchor/android/fameofshame/StartTimerActivity.java

```diff
@@ -0,0 +1,47 @@
+package com.kaamchor.android.fameofshame;
+
+import android.os.Bundle;
+import android.widget.NumberPicker;
+import android.widget.TextView;
+import android.widget.TextView;
+
+/**
+ * Created by hanandarugar on 26/12/17.
+ */
+
+
+public class StartTimerActivity extends CountDownActivity {
+
+    private TextView Hour, Minute,Second;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_start_timer);
+        initTimer();
+        countDownStart();
+    }
+
+    public void initTimer(){
+        Hour = (TextView) findViewById(R.id.hourTimer);
+        Minute = (TextView) findViewById(R.id.minsTimer);
+        Second = (TextView) findViewById(R.id.secTimer);
+    }
+
+    public void countDownStart() {
+        int hr = getHour();
```

Comment:
instead of having getters, pass the value in the intent while starting the activity